diff --git a/node_modules/@remix-run/react/components.js b/node_modules/@remix-run/react/components.js
index 49a3805..4795685 100644
--- a/node_modules/@remix-run/react/components.js
+++ b/node_modules/@remix-run/react/components.js
@@ -533,11 +533,18 @@ function Meta() {
     parentsData[routeId] = data;
   }
 
+  let [ogVertical] = (
+    typeof meta["og:type"] === "string" ? meta["og:type"] : ""
+  ).split(".");
+
   return /*#__PURE__*/React__namespace.createElement(React__namespace.Fragment, null, Object.entries(meta).map(([name, value]) => {
     if (!value) return null; // Open Graph tags use the `property` attribute, while other meta tags
     // use `name`. See https://ogp.me/
 
-    let isOpenGraphTag = name.startsWith("og:");
+    let isOpenGraphTag = name.startsWith("og:") ||
+      (ogVertical && name.startsWith(`${ogVertical}:`));
+
+
     return name === "title" ? /*#__PURE__*/React__namespace.createElement("title", {
       key: "title"
     }, value) : ["charset", "charSet"].includes(name) ? /*#__PURE__*/React__namespace.createElement("meta", {
